### PR TITLE
Bump to v1.8.2

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

File: contrib/spec/podman.spec.in [hunk 1]
```diff
@@ -48,7 +48,7 @@ Epoch: 99
 %else
 Epoch: 0
 %endif
-Version: 1.8.2
+Version: 1.8.3
```
Comment (TomSweeneyRedHat) [inline contrib/spec/podman.spec.in]: shouldn't this be 1.8.3-dev?

Comment (mheon) [reply contrib/spec/podman.spec.in]: Not for the RPM, we don't append `-rc` there

**Comment by mheon** [comment]
Lots of registry flakes, restarted all of them

**Pull request merged**